src/System.Net.Http.WinHttpHandler/tests/FunctionalTests/WinHttpHandlerTest.cs

```diff
@@ -107,6 +110,24 @@ public void SendAsync_SlowServerRespondsAfterDefaultReceiveTimeout_ThrowsHttpReq
             }
         }
 
+        [ConditionalTheory(nameof(IsWindows10Version1607)), MemberData(nameof(Http2Servers))]
```

Comment:
I will move this new test to the System.Net.Http.Functional.Tests project.  A future PR can help consolidate the HTTP/2 tests.  The important thing is that this test makes sure to verify that HTTP/2 is actually working end-to-end.